portfolio/src/main/java/com/google/sps/data/Comments.java

```diff
@@ -0,0 +1,38 @@
+package com.google.sps.data;
+
+import java.util.Date;
```

Comment:
nit: prefer java.time lib over java.util.Date. see go/javatime

Reply:
I dont have access to go/javatime :(
But ill look into the documentation and consider changing my implementation later on. For now can I keep Date, since our walkthrough uses it?

Reply:
if the walkthrough uses it then it should be fine. No need to change it.

Reply:
Cant acces go/javatime :(
The walkthrough contains Date, I will consider changing implementation in the future, but since its easy to interchange can I keep it as Date for now and then read the documentation for time later?